### PR TITLE
Run benchmarks via script

### LINKED ISSUE #4: Drop make usage for test runs
It may be useful for builds (but that is not obvious for this particular case), but uncommenting something to run the test is just plain wrong, we need some trivial `./run.sh -with -some -flags` to drive test execution.

### PR DESCRIPTION
**Pull request by fyrchik**
1. Dockerfile fixes: Unify format and actually use declared network. 
2. Move all parameter hanling in `runner.sh`.

Close #4.

### CODE REVIEW

File: .docker/ir/docker-compose.sharp.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 version: '2.4'
 
+networks:
+  default:
+    name: neo_go_network
+    ipam:
+      config:
+        - subnet: 172.200.0.0/24
```
Comment (roman-khimov) [inline .docker/ir/docker-compose.sharp.yml:8]: It's gonna use the same subnet for every run?

Comment (fyrchik) [reply .docker/ir/docker-compose.sharp.yml:8]: Yes, but this is not a problem unless runs are performed in parallel (they can't currently).

Comment (roman-khimov) [reply .docker/ir/docker-compose.sharp.yml:8]: Actually, that's good, makes the system more predictable, knowing exactly where to knock at with pprof request is already a big improvement.

**Comment by fyrchik** [comment]
So with last commit I have tried to make sensible defaults so that default invocation is just `./runner.sh -w 30`.
We can even try to allow providing empty list of arguments but this needs some refactoring.

File: .docker/ir/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+LOGGING=none
```
[the remark this answers is not in the document]
Comment (fyrchik) [reply .docker/ir/.env]: I think this is already documented in help, but we still have ability to use environment, so I also created `ENV.md`.
Fixed.

[the remark this answers is not in the document]
Comment (fyrchik) [reply .docker/ir/.env]: Moved to `README.md`.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**